configs/vision/pathology/offline/classification/panda.yaml

```diff
@@ -94,7 +94,7 @@ data:
       predict:
         - class_path: eva.vision.datasets.PANDA
           init_args: &PREDICT_DATASET_ARGS
-            root: ${oc.env:DATA_ROOT, ./data/panda/prostate-cancer-grade-assessment}
+            root: ${oc.env:DATA_ROOT, ./data/panda}
```

Comment:
if you download the dataset from kaggle it will be in `panda/prostate-cancer-grade-assessment`. So might be better to leave the full path as default

Reply:
Good point, updated now.